code/game/objects/items/bodybag.dm

```diff
@@ -77,8 +77,13 @@
 	user.last_special = world.time + CLICK_CD_BREAKOUT
 	to_chat(user, "<span class='notice'>You claw at the fabric of [src], trying to tear it open...</span>")
 	to_chat(loc, "<span class='warning'>Someone starts trying to break free of [src]!</span>")
-	if(!do_after(user, 200, target = src))
-		to_chat(loc, "<span class='warning'>The pressure subsides. It seems that they've stopped resisting...</span>")
-		return
-	loc.visible_message("<span class='warning'>[user] suddenly appears in front of [loc]!</span>", "<span class='userdanger'>[user] breaks free of [src]!</span>")
-	qdel(src)
+	if(do_after_mob(user, src, 12 SECONDS, TRUE))
```

Comment:
```suggestion
	if(!isliving(src.loc) || do_after(user, 10 SECONDS, target = src))
```
This way, you can still use it for kidnapping, but you can't cremate someone with no possible counterplay for them to escape (aside from, say, putting the bluespace body bag in the hands of a monkeyman or something, which you could maybe solve with a client check).

Reply:
Oh, and I guess maybe a check to keep people from just chucking a bluespace body bag that contains someone into a recycler would also be nice.

Reply:
it doesnt make sense to me that being trapped several layers in somehow makes it easier to escape than being trapped while in someones hands.